src/index.html

```diff
@@ -1,22 +1,36 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo" class="logo"><img src="../img/moyo_1.png" alt="img"></a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_list-item is-active">
+            <a class="nav_link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav_list-item"><a href="#" class="nav_link">Samsung</a></li>
```

Comment:
Fix code style everywhere and add the blank line between the links
```suggestion
          </li>
          <li class="nav_list-item"><a href="#" class="nav_link">Samsung</a></li>
```
```suggestion
          </li>
          
          <li class="nav_list-item">
            <a href="#" class="nav_link">Samsung</a>
          </li>
```